src/models/Networktype.ts

```diff
@@ -0,0 +1,4 @@
+
+export const supportedNetworks = ["alphanet", "testnet", "shimmer", "iota"] as const;
```

Comment:
These names should be populated by a config file instead of being static in code.